wpeverest/user-registration
# UR-1047 Tweak - Form Preview

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .

### How to test the changes in this Pull Request:

1.Preview form from form builder
2.verify if apply theme is working  and sync with frontend
3.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Form Preview
